### ISSUE TITLE
Add Different Visualizations in the Top Bar

### ISSUE THREAD

**Issue by schrej** [fflewddur/tophat#63]
Nice to finally have an extension with a nice UI.

One thing I'm missing that other plugins (and the great iStats Menu for macOS) have is the option to switch between different Visualisations. I can think of the following

- Bar(s) (currently used for cpu and memory)
- Graph (cpu, network)
- Pie Chart (memory)
- Percentage (all)
- Absolute Numbers (currently used for network, also interesting for memory)

It might make sense to allow using both a graphical and a numerical representation at the same time, e.g. by having two dropdowns that allow to select one option for graphical, and one for textual representation with the option to select nothing for either.

I would personally use a graph for CPU, a pie chart for memory and numbers (or a graph) for network.

**Comment by fgvieira** [comment]
That would be great! Not sure what you mean by "Graph", but I'd really like to see stats over the last (e.g.) 5 minutes (like in https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet)

**Comment by lap1dem** [comment]
Copy that suggestion! It would be great to have pie charts for memory and disk usage. As a personal preference - I would prefer a pie chart with a hole in the centre, so it would look more like a circular progress bar

**Comment by fflewddur** [comment]
Yep, I agree different visualization options would be nice to have. In the near term I'd like to get text labels working as an option for all of the monitors, and also try a dual horizontal bar layout for network and disk activity, with one bar for write/send activity and another for read/receive activity.
